src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
 // write your code here
+const h1 = document.querySelector('.logo');
```

Comment:
You need to create two promises, `promise1` and `promise2`, as specified in the task description. Currently, you are directly using functions `promSuccess` and `promError` without creating these promises.